Userland/Libraries/LibJS/Runtime/ArrayBuffer.h

```diff
@@ -170,12 +170,13 @@ inline size_t array_buffer_byte_length(ArrayBuffer const& array_buffer, ArrayBuf
     return array_buffer.byte_length();
 }
 
-// 25.1.3.14 RawBytesToNumeric ( type, rawBytes, isLittleEndian ), https://tc39.es/ecma262/#sec-rawbytestonumeric
+// 25.1.3.14 RawBytesToNumeric ( type, rawBytes, isLittleEndian )
+// Using stage three proposal for Float16Arrays: https://tc39.es/proposal-float16array/#sec-rawbytestonumeric
```

Comment:
Fly-by comment, we don't delete the mainline spec link when adding proposal changes. We also don't need to write the stage number. So this comment, and the similar ones below, should all be of the form:
```c++
// 25.1.3.14 RawBytesToNumeric ( type, rawBytes, isLittleEndian ), https://tc39.es/ecma262/#sec-rawbytestonumeric
// 5 RawBytesToNumeric ( type, rawBytes, isLittleEndian ), https://tc39.es/proposal-float16array/#sec-rawbytestonumeric
```